easybuild/framework/easyblock.py

```diff
@@ -1445,7 +1445,13 @@ def extensions_step(self, fetch=False):
 
     def package_step(self):
         """Package software (e.g. into an RPM)."""
-        pass
+        rpmname = "nxprog-%s-%s" % (self.name, self.version)
```

Comment:
@sntgluca : nxprog looks like a hardwired prefix; can we make it generic by picking it up from a configuration option?